TOC.md

```diff
@@ -7,6 +7,7 @@
 
 + Introduction
   - [TiDB Introduction](/overview.md)
+  - [Quick Start with TiDB](/how-to/get-started/quick-start-with-tidb.md)
```

Comment:
```suggestion
  - [Quick Start Guide for the TiDB Database Platform](/how-to/get-started/quick-start-with-tidb.md)
```
Do we need to move the file location as the ToC suggests? @queenypingcap

Reply:
Yes, please do

Reply:
Please change the file location and the ToC link, thx~ @ran-huang

Reply:
What do you think of moving it one level up to be in parallel with Introduction? It may not make sense if you think it strictly from how it should be categorized, but it is the most important information a reader want to read.

Reply:
I might consider it a good trade-off. It's worth trying at least for some time. If anyone poses a challenge to the file location, we can re-discuss it with more feedback then.

Reply:
Updated. PTAL~@queenypingcap @yikeke